meinberlin/apps/account/templates/meinberlin_account/actions.html

```diff
@@ -6,6 +6,7 @@
 
 {% block dashboard_content %}
     <h1>{% trans 'Activities' %}</h1>
+    <p>{% trans 'Here you can find any actions that have happened in projects you are following.' %}</p>
```

Comment:
I personally would prefer "This is a list of actions" instead of "Here you can find any actions"